tests/codegen/license.cpp

```diff
@@ -7,8 +7,7 @@ namespace codegen {
 TEST(codegen, license)
 {
   auto bpftrace = get_mock_bpftrace();
-  auto configs = ConfigSetter(*bpftrace->config_, ConfigSource::script);
-  configs.set(ConfigKeyString::license, "Dual BSD/GPL");
+  bpftrace->config_->license = "Dual BSD/GPL";
```

Comment:
I have to admit this kind of setting is one thing I liked about the old Config class implementation - being able to accidentally do this on non-test code paths is a bit sad 😢

Reply:
There are language features for this, we should be passing around a `const Config&` to places that don't need mutability :)

Reply:
That's fair. I guess there really is only one or two places that needs to mutate the Config object.